client/tso_client.go

```diff
@@ -89,15 +93,26 @@ func (c *client) GetLocalTSWithinKeyspaceAsync(ctx context.Context, dcLocation s
 	return req
 }
 
-var _ BaseClient = (*tsoBaseClient)(nil)
+const (
+	// tsoKeyspaceGroupPrimaryElectionPrefix defines the key prefix for keyspace group primary election.
+	// The entire key is in the format of "/pd/<cluster-id>/microservice/tso/keyspace-group-XXXXX/primary" in which
+	// XXXXX is 5 digits integer with leading zeros. For now we use 0 as the default cluster id.
+	tsoKeyspaceGroupPrimaryElectionPrefix = "/pd/0/microservice/tso/keyspace-group-"
```

Comment:
I noticed that there is a comment left previously. See https://github.com/tikv/pd/pull/6008#discussion_r1118504891

Reply:
Yes, we can certainly move "keyspace-group-" from the path to make it more compact. I'll make the change.